package-list.json

```diff
@@ -13,7 +13,7 @@
     },
     {
       "version": "2.1.0",
-      "desc": "Pre-STU3 development version of mCODE",
+      "desc": "STU 2 update with new SNOMED codes",
```

Comment:
THis one puzzled me....isn't that STU2.1?
The former description seems more accurate.